bin/run-addon

```diff
@@ -9,20 +9,20 @@ jpm="$base/node_modules/.bin/jpm"
 nodemon_pid=
 
 binary=
-firefoxes="
-/Applications/FirefoxNightly.app
-/Applications/Firefox\ Nightly.app
-/Applications/FirefoxDeveloperEdition.app
-/Applications/Firefox\ Developer\ Edition.app
-/Applications/FirefoxAurora.app
-$(which firefox || true)
-"
+firefoxes=(
+"/Applications/FirefoxNightly.app"
+"/Applications/Firefox Nightly.app"
+"/Applications/FirefoxDeveloperEdition.app"
+"/Applications/Firefox Developer Edition.app"
+"/Applications/FirefoxAurora.app"
+"$(which firefox || true)"
+)
```

Comment:
It probably makes more sense to have `firefox` first?  That way I can change wherever `firefox` is symlink'd to run a different version of Firefox.